src/App.js

```diff
@@ -1,16 +1,141 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
 
-class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+const todoListArray = [
+  {
+    task: 'Learn React',
+    id: 1528817077286,
+    completed: false
+  },
+
+  {
+    task: 'Dont go crazy',
+    id: 1528817084358,
+    completed: false
+  },
+
+  {
+    task: 'Walk Jolie',
+    id: 1528817027272,
+    completed: false
+  }
+];
+
+
+
+class App extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      todoList: todoListArray,
+      todoText: '',
+    };
+  }
+
+  handleChange = event => {
+    // console.log('event', event.target);
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  updateList = event => {
+    event.preventDefault();
+
+    this.setState({
+      todoList: [
+        ...this.state.todoList,
+        {task: this.state.todoText,
+        id: Date.now(),
+        completed: false,}
+      ],
+
+      todoText: '',
+    });
+
+  }
+
+  // finished = id => {
+  //   let newTodoList = this.state.todoList.slice();
+  //   newTodoList = newTodoList.map(item => {
+  //     if(item.id === id) {
+  //       item.completed = !item.completed;
+  //       return item;
+  //     }
+
+  //     else {
+  //       return item;
+  //     }
+  //   });
+
+  //   this.setState({
+  //     todoList: newTodoList
+  //   });
+  // }
+
+  // both version of finished work.
+  // trying to simplify it so it makes more sense to me.
```

Comment:
awesome! always find new ways to do the same thing, a creative mindset will work wonders for you.